responder/api.py

```diff
@@ -65,7 +65,7 @@ def __init__(
 
         if static_dir is not None:
             if static_route is None:
-                static_route = static_dir
+                static_route = ""
```

Comment:
Don't use a filesystem path as an URL path.

Reply:
Thanks for catching this.